### ISSUE TITLE
HIVE Y3 feature notepad

### ISSUE THREAD

**Issue by LibbyUX** [hubmapconsortium/hra-ui#545]
Katy provides her meeting notes from a meeting with Ajay

# US#1 CTann

Trusted repository, # tissue samples.
Which of these rows are more accurate
Make conservative predictions
How far is real data from predicted data
#datasets used for a prediction
#labs that 

# US#1 Predict 3D location
Aim to read in/support standard sc analysis outputs
Read in h5ad files

Show result in EUI

1:17 datasets are ALWAYS partial, truncated.
What happens if partial data is uploaded?
Consider rejecting data with low cell counts, too few CTs.
If randomly sampled data is missing then we might still have a correct CTpop--just half of the cells per CT.
Get low quality datasets from TMCs and see how much error results.
Fake datasets, add random noise
Add info on how much data is used to make a prediction.
Test this with Anatomist

Ajay can understand ‘Similar Dataset’

# US#3 iFTU

Donor characteristics
10x, smartseq

 Ajay asks Bruce to look into EPICS--to access FTUs and sc identified by TMC.
The automation via HIVE might just not happen.

# US#3 CDE
How would one know what nodes are endothelial (begin or end of line?) - color
Yes, green is most dominant

![image](https://github.com/hubmapconsortium/hra-ui/assets/97060856/38d0dc1e-ba73-4344-9fd4-104f0964b894)

Show H&E slide also so people see pathology

Lines are useful but summaries/graphs are even more useful.
Tumor cells should have more dense blood supply. Not sure we see this in network layout.

Add violin plot in MVP2.
Need to ‘extract biology’.

Density metrics would be valuable.

# US#6 HRA Components

Make sure people understand what a biologist can take this code

![image](https://github.com/hubmapconsortium/hra-ui/assets/97060856/61492847-9566-4d39-89ab-8349223a15bc)

Save as a html and have this functionality!

Need more descriptive titles. ‘Use Atlas Components’ is not good.

# US#7 HRA Dashboard
HRA KG is NOT meaningful - are the nodes? Are they genes, CTs?
Improve (i)!!!!!!!!

Make sure users understand we did not use 7M publications for the HRA.
Leave out pubs? Only feature HRA pubs?

Use running average for
![image](https://github.com/hubmapconsortium/hra-ui/assets/97060856/fefd2c8e-38f1-4dfb-812f-9ecc9506e89b)

Web request has info about referrer.
EBI is OLS with 3D organ.

Add this info to (i)

**Comment by LibbyUX** [comment]
# Cell Distance Explorer MVP2 Feasibility Check

Good morning @bherr2 & @J-Yash,

Bruce, Yash and I met last night to mark up my current wireframes and ideate on features we could add to MVP2. I reviewed our [March usability study notes](https://www.figma.com/board/Azw0jLJH3Cfd5tlM3iX95x/UX-Studies?node-id=755-624&t=l8QaXgpbHbRcvrVj-4) this morning, along with the many notes we have floating around here in GitHub. Lots to feasibility check with you first before I design for handoff on August 30th:

## CDE Potential features for now or future versions:
### Vis App Features
1. All vis/plot modules: Full screen modal overlay for visualization & plots
    - Users can tab between two views in this modal: visualization or plots
    - Users can close out of full screen with an "X" close icon button in the top left to return to the regular vis app
2. Visualization module: Lock the scale bar to the bottom right corner of visualization modal
3. Cell Types Module: Option to see listing of relationships/edges.
    a. Cell Types Module: Option to hide/show relationships/edges just like cell types. 
    b. Cell Types Module: Option to edit color of relationships/edges.
4. Visualization module: Hover card on cells/nodes to receive cell ID, all available coordinates, distance to closest anchor cell.
5. Visualization module: Lasso freeform drawing selection tool. The ability to draw in the visualization, select, deselect, show, and hide certain areas for analysis.
6. Downloads: Add headers to cells/nodes and relationships/edges downloads.

### Create Vis Features
1. Update optional anchor cell type step to required "Configure Parameters" step. Users by default will have the distance threshold be 1000 and the pixel size be 1. We will compute the update on our end if they customize. @J-Yash may need to clarify expectations further for this feature.
    - It is required, but we have preselected the following features:


### Analysis Workflow: Create Vis to Vis App Features
1. Upload second dataset to see comparison in violin plot. Per [your comment in Figma](https://www.figma.com/design/G0xVFFsAAnBwzHd8PtQEeb?node-id=2704-25852#908983991), @bherr2, we will aim for MVP3.
2. Upload image in Create Vis/Vis App and use opacity to compare images. Yash warned me that this could be a BIG file size.
3. Use user uploaded CL ID data to (see [ongoing comment thread in Figma](https://www.figma.com/design/G0xVFFsAAnBwzHd8PtQEeb?node-id=2652-25287#909003011)):
    a. Report CL ID on hover in the Vis Apps Cell List module
    b. Switch between cell name and CL ID
    c. Consolidate similar cell types into parents/condense low level cells.
    d. Group similar cell types.
    e. Report CL ID on hover of a cell/node in the Visualization module

## Features currently being specified
### Vis App
- Violin plot module #627 
- Mat-menu with "More" icon entry point
- Embed modal for embedding the application
- Per Katy: Remove legend from Histogram module and add all cells to top of Cell Types module
### Create Vis
- Configurable column headers for improved interoperability between cell data analysis workflows #579 

## Resources
- Figma canvas: [CDE MVP2](https://www.figma.com/design/G0xVFFsAAnBwzHd8PtQEeb/Cell-Distance-Explorer?node-id=2651-44436&t=mnuS3gMRp2Firnlz-4)

**Comment by bherr2** [comment]
`Upload image in Create Vis/Vis App and use opacity to compare images. Yash warned me that this could be a BIG file size.` is going to be out of scope for MVP2. It will definitely be too much to support in a short time.

**Comment by LibbyUX** [comment]
@bherr2 thank you! Should I design everything else listed?

**Comment by bherr2** [comment]
Yep! But to be clear, some things may still need to get cut based on timing. We need to see how much time we'll actually have once the Spec is closer to completion and we have a better idea of the timing.
